### ISSUE TITLE
Android After appcompat-v7:23.0.1 upgrading, getStatus().getStatusCode() is not used.

### ISSUE THREAD

**Issue by olkunmustafa** [Azure/azure-mobile-services#798]
In Android, After appcombat library is upgraded to `appcompat-v7:23.0.1` from `appcompat-v7:22.2.1` , the code that is used for get status from Azure Backend is not working.

```int responseCode = mEx.getResponse().getStatus().getStatusCode();```

What is the alternative way to get the status code from Azure Backend Service.

<img width="527" alt="screen shot 2015-09-18 at 17 08 47" src="https://cloud.githubusercontent.com/assets/5277158/9961292/6447cada-5e28-11e5-93e1-6a80960f0445.png">

Thanks.

**Comment by soninaren** [comment]
We are tracking this issue here #776. The fix is in our backlog and should be out soon. #776 has a work around posted that would help you unblock.

**Comment by paulbatum** [comment]
Duplicate of #776
